Mathlib/Topology/Algebra/Valuation.lean

```diff
@@ -166,4 +166,24 @@ theorem cauchy_iff {F : Filter R} : Cauchy F ↔
     exact h γ
 #align valued.cauchy_iff Valued.cauchy_iff
 
+variable (R)
+
+/-- The unit ball of a valued ring is open. -/
+theorem integer_isOpen :
+    IsOpen (_i.v.integer : Set R) := by
```

Comment:
Not that it matters but I'd be tempted to just put the statement on one line here

Reply:
Maybe we can rename the variable `_i` too, it looks very strange using that in the statement of a theorem